server/sonar-server/src/test/resources/org/sonar/server/issue/ws/SearchActionMediumTest/filter_by_assigned_to_me.json

```diff
@@ -0,0 +1,42 @@
+{
+  "issues": [
+    {
+      "key": "82fd47d4-b650-4037-80bc-7b112bd4eac2",
+      "component": "MyComponent",
+      "project": "MyProject",
+      "rule": "xoo:x1",
+      "status": "OPEN",
+      "severity": "MAJOR",
+      "debt": "10min",
+      "fUpdateAge": "less than a minute",
+      "assignee": "john"
+    }
+  ],
+  "facets": [
+    {
+      "property": "assignees",
+      "values": [
+        {
+          "val": "john",
+          "count": 1
+        },
+        {
+          "val": "alice",
+          "count": 1
+        },
+        {
+          "val": "",
```

Comment:
I imagine that this empty value is because of the "assigned to me". Would it be possible to remove this entry ?